components/map/events.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Events
+page_title: Map - Events
+description: Discover the Blazor Map events and explore the examples.
+slug: components/map/events
+tags: telerik,blazor,map,events,event
+published: true
+position: 11
+---
+
+# Map Events
+
+This article explains the available events for the Telerik Map for Blazor:
+
+* [OnClick](#onclick)
+* [OnMarkerClick](#onmarkerclick)
+* [OnShapeClick](#onshapeclick)
+
+>important The **asynchronous** rendering is **not** supported.
+
+## OnClick
+
+The `OnClick` event fires as a response to the user click on the map. Its `EventCallback<MapClickEventArgs>` gives:
+* `MapClickEventArgs.EventArgs` - provides the native DOM event (browser event).
+* `MapClickEventArgs.Location` - provides the location of the click on the map (`MapLocation` has `Latitude` and `Longitude` props).
+
+>caption Handle OnClick.
+
+````CSHTML
+@* This code snippet showcases an example of how to handle the OnClick event. *@
+
+<TelerikMap Center="@Center"
+            Zoom="3"
+            OnClick="@OnMapClick">
+    <MapLayers>
+        <MapLayer Type="@MapLayersType.Tile"
+                  Attribution="@Attribution"
+                  Subdomains="@Subdomains"
+                  UrlTemplate="@UrlTemplate">
+        </MapLayer>
+
+        <MapLayer Type="@MapLayersType.Bubble"
+                  Data="@BubbleData"
+                  LocationField="@nameof(BubbleModel.LatLng)"
+                  ValueField="@nameof(BubbleModel.Revenue)">
+            <MapLayerBubbleSettings>
+                <MapLayerBubbleSettingsStyle>
+                    <MapLayerBubbleSettingsStyleFill Color="#0000ff"></MapLayerBubbleSettingsStyleFill>
+                    <MapLayerBubbleSettingsStyleStroke Color="#000000"></MapLayerBubbleSettingsStyleStroke>
+                </MapLayerBubbleSettingsStyle>
+            </MapLayerBubbleSettings>
+        </MapLayer>
+
+        <MapLayer Type="@MapLayersType.Marker"
+                  Data="@MarkerData1"
+                  LocationField="@nameof(MarkerModel.LatLng)"
+                  TitleField="@nameof(MarkerModel.Title)">
+        </MapLayer>
```

Comment:
Not sure if we want to use such a detailed configuration just to show the click event. I see that it is being reused across all event examples, so this might have been the general idea behind it.

Reply:
Yep, the general idea is to use the same configuration with a different event. This will make the user more easily distinguish all the clickable places on the map.